docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -49,7 +49,7 @@ deactivate AddressBookParser
 LogicManager -> DeleteCommand : execute(m)
 activate DeleteCommand
 
```

Comment:
Since we are passing in an array of Indexes, I think the command here could specify it a bit clearer, like `deletePerson([1, 2, 3])`